src/components/CategorySelector.tsx

```diff
@@ -93,19 +96,40 @@ export const CategorySelector = ({
           InputProps={{ endAdornment: <SearchIcon /> }}
           onChange={(event) => setSearchValue(event.target.value)}
         />
-        <Box
-          sx={{
-            display: 'flex',
-            gap: '0.5rem',
-            alignItems: 'center',
-          }}>
-          <FilterVintageIcon
-            color="primary"
-            titleAccess={t('registration.resource_type.nvi.can_give_publication_points')}
-            fontSize="small"
-          />
-          <Typography>{t('registration.resource_type.nvi.can_give_publication_points')}</Typography>
-        </Box>
+        {enableNviHighLightning ? (
+          <Box sx={{ display: 'flex', gap: '0.5rem', alignItems: 'center' }}>
+            <Chip
+              icon={
+                <FilterVintageIcon
+                  color="primary"
+                  titleAccess={t('registration.resource_type.nvi.can_give_publication_points')}
+                  fontSize="small"
+                />
+              }
+              color="primary"
+              onClick={() => {
+                setHighlightNviCategories(!higlightNviCategories);
+              }}
```

Comment:
Jeg foretrekker one-liner når det kun er ett funksjonskall. Men det er min subjektive mening, så opp til deg :) 

```suggestion
              onClick={() => setHighlightNviCategories(!higlightNviCategories)}
```